html/browser/common.css

```diff
@@ -34,6 +34,10 @@ a, button, a:link, a:visited, a:active, .linkOn, .linkOff
 	user-select: none;
 }
 
+.dmIcon
+{
+	-ms-interpolation-mode: nearest-neighbor;
+}
```

Comment:
this is already done in Image, on by default. if you want to turn it off use `<Image fixBlur={false}`